__tests__/seed.test.js

```diff
@@ -114,31 +119,33 @@ describe("GET /api/articles/:article_id/comments", () => {
         expect(allComments.length).toBe(11);
         expect(allComments).toBeSortedBy("created_at", { descending: true });
         allComments.forEach((comment) => {
-          expect(typeof comment.comment_id).toBe("number");
-          expect(typeof comment.votes).toBe("number");
-          expect(typeof comment.created_at).toBe("string");
-          expect(typeof comment.author).toBe("string");
-          expect(typeof comment.body).toBe("string");
-          expect(comment.article_id).toBe(1);
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
```

Comment:
you can also use any(Date) here for a bit of extra security

Reply:
I tired doing any(Date), however it came back with error, think it's something to do with the utils functions? I did ended up passing it using 'expect.toBeDateString()' :)

Reply:
ah interesting! that works equally as good then nice catch